ui/src/timeMachine/reducers/index.ts

```diff
@@ -829,10 +836,13 @@ export const timeMachineReducer = (
         const {index} = action.payload
         const draftQuery = draftState.draftQueries[draftState.activeQueryIndex]
 
-        const selectedValues = draftQuery.builderConfig.tags[index].values
+        let selectedValues = []
+        if (draftQuery) {
+          selectedValues = draftQuery.builderConfig.tags[index].values
 
-        draftQuery.builderConfig.tags.splice(index, 1)
-        draftState.queryBuilder.tags.splice(index, 1)
+          draftQuery.builderConfig.tags.splice(index, 1)
+          draftState.queryBuilder.tags.splice(index, 1)
+        }
```

Comment:
A reducer test for this added logic would be 👍

Reply:
@121watts this change i made only adds a check that `draftState` exists. I'll add tests for this, but in a different ticket since this is kind of big and needs to get out.

https://github.com/influxdata/influxdb/issues/16294

Reply:
@121watts this change i made only adds a check that `draftState` exists. I'll add tests for this, but in a different ticket since this is kind of big and needs to get out.

https://github.com/influxdata/influxdb/issues/16294